src/store/__mocks__/RelayStore.js

```diff
@@ -79,11 +83,26 @@ RelayStore.forceFetch.mockImplementation((...args) => {
   return returnValue;
 });
 
-RelayStore.mock = {
-  setMockRecords: records => {
-    RelayStore.mock.recordStore = new RelayRecordStore({records});
-  },
-  recordStore: null,
-};
+RelayStore.hasOptimisticUpdate.mockImplementation((dataID) => {
+  return storeData.hasOptimisticUpdate(dataID);
+});
+
+RelayStore.resolve.mockImplementation((...args) => {
+  return new GraphQLStoreQueryResolver(storeData, ...args);
+});
+
+RelayStore.buildDeferredFragmentState.mockImplementation((...args) => {
+  return new RelayDeferredFragmentState(
```

Comment:
I don't like how these are just reimplementations... maybe this could become a `requireActual` that then just hooks on the mocking functionality that some of the tests rely on.